doc/apps/dgst.pod

```diff
@@ -14,21 +14,24 @@ B<openssl> B<dgst>
 [B<-binary>]
 [B<-r>]
 [B<-non-fips-allow>]
-[B<-out filename>]
-[B<-sign filename>]
-[B<-keyform arg>]
-[B<-passin arg>]
-[B<-verify filename>]
-[B<-prverify filename>]
-[B<-signature filename>]
-[B<-hmac key>]
+[B<-out> I<filename>]
+[B<-sign> I<filename>]
+[B<-keyform> I<arg>]
+[B<-passin> I<arg>]
+[B<-verify> I<filename>]
+[B<-prverify> I<filename>]
+[B<-signature> I<filename>]
+[B<-hmac> I<key>]
+[B<-mac> I<alg>]
+[B<-macopt> I<nm>B<:>I<v> ...]
+[B<-rand> I<files>]
 [B<-non-fips-allow>]
 [B<-fips-fingerprint>]
-[B<file...>]
+[I<file ...>]
```

Comment:
I'm not sure which way this should be. In cases where it's a mixture of replaceable text and literal or structural arguments, the '...' is like a structural element, and should probably be plain. In this case, it's repeating just user-replaceable elements so I could see an argument for either way.

The BSD man pages tend to underline their '...'s in cases like this.

```
[I<file ...>]
```

<img width="533" alt="screen shot 2015-09-17 at 1 27 12 pm" src="https://cloud.githubusercontent.com/assets/2618447/9940813/e778f3be-5d3f-11e5-9a51-0987b64bd8d2.png">

<img width="452" alt="screen shot 2015-09-17 at 1 28 20 pm" src="https://cloud.githubusercontent.com/assets/2618447/9940830/fd38201c-5d3f-11e5-8817-8d79e4e60437.png">

So does `man man`, which I think should be given weight.

<img width="560" alt="screen shot 2015-09-17 at 1 29 09 pm" src="https://cloud.githubusercontent.com/assets/2618447/9940850/19b318aa-5d40-11e5-8fbd-718da82e6a51.png">

But `man groff` leaves them plain.

```
[I<file> ...]
```
<img width="556" alt="screen shot 2015-09-17 at 1 39 39 pm" src="https://cloud.githubusercontent.com/assets/2618447/9941185/a0d3879c-5d41-11e5-87d6-42f86858254b.png">



GNU tends to leave them plain, and also omits the space between the '...' and the thing to be repeated, and put the '...' outside syntax structural markup. This is probably more formally correct, but I find it less readable. (Also, GNU hates `man`, so maybe take them with a grain of salt.)

```
[I<file>...]
```

<img width="392" alt="screen shot 2015-09-17 at 1 30 50 pm" src="https://cloud.githubusercontent.com/assets/2618447/9940896/540785ea-5d40-11e5-9e98-8cb41042e5a7.png">

LLVM underlines and omits spaces.

```
[I<file...>]
```

<img width="511" alt="screen shot 2015-09-17 at 1 31 32 pm" src="https://cloud.githubusercontent.com/assets/2618447/9940936/898db7c0-5d40-11e5-8893-18efceb81747.png">

X11 is inconsistent.

So, I think it could go either way. Personally, I find the underlined form to be more readable, since it's more user-replaceable text like <u>`file`</u> is.

Anyone else have thoughts or preferences?